include/galsim/SBInclinedExponential.h

```diff
@@ -69,9 +69,12 @@ namespace galsim {
         /// @brief Returns the scale radius r0 of the disk profile
         double getScaleRadius() const;
 
-        /// @brief Returns the scale height r0 of the disk profile
+        /// @brief Returns the scale height h0 of the disk profile
         double getScaleHeight() const;
 
+        /// @brief Returns the scale height over scale radius h0/r0 of the disk profile
+        double getScaleHOverR() const;
```

Comment:
I can't imagine any situation where the efficiency of the `getScaleHOverR` is at all important.  In fact, this implementation is probably slower, since you _always_ calculate h/r in the constructor to store the value, even if it is not ever accessed.  Unless users will generally access this multiple times, that's not going to be faster overall.  (Although the slow down is, of course, minuscule...)

I'm also not convinced that the user will ever care about retrieving h/r from an existing object.  But I'll leave the final decision on that up to you.